src/VerletList.py

```diff
@@ -25,14 +25,18 @@
 *********************
 
 
-.. function:: espressopp.VerletList(system, cutoff, exclusionlist)
+.. function:: espressopp.VerletList(system, cutoff, exclusionlist, useBuffers, useSOA)
 
 		:param system: 
 		:param cutoff: 
 		:param exclusionlist: (default: [])
+		:param useBuffers: Whether particle neighbors are buffered to improve rebuild times. (default: False)
```

Comment:
:thinking: in the constructor `useBuffers = true` and here `(default: False)`

Reply:
Sorry, they were all initially False. Now I've set everything to True by default including in the C++ class.